test/helpers/redis.ts

```diff
@@ -19,3 +23,43 @@ export const clearLimitersWithExclusions = async (exclusions: string[]) => {
 }
 
 export const clearLimiters = () => clearLimitersWithExclusions([])
+
+let ioRedis
+let pricePublishInterval
+export const publishOkexPrice = () => {
+  ioRedis = new Redis(6379, process.env.REDIS_0_INTERNAL_IP || "localhost")
+  pricePublishInterval = setInterval(() => {
+    const timestamp = Math.floor(new Date().getTime() / 1000)
+    const message = {
+      meta: {
+        publishedAt: timestamp,
+        correlationId: crypto.randomUUID(),
```

Comment:
`CorrelationId` is a pattern by which you can link together discrete process steps that don't necessarily have a parent / child relationship (like when using `trace_id` / `span_id`).

In the Stablesats example it makes it possible to tie together the receiving of the price update from okex with the call to the price server.

see this query for instance https://ui.honeycomb.io/galoy/datasets/jc-stablesats-dev/result/dScQBZGeetV

Reply:
seems the [opentelemetry convention](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/messaging/) for that is `conversation_id`, but I find `message_id` more meaningful. 

anyway feels free to keep the name if you prefer it like that

Reply:
Thanks for the link - very useful article!

A few things to note:
- this is a field in the message body (ie managed by our own code) and not metadata automatically added by the pubsub system. 
- `message_id` in the convention doc is defined as `A value used by the messaging system as an identifier for the message, represented as a string.` Redis (unlike their examples of Kafka and RabbitMQ) doesn't have such a message id.
- If this were purely for message passing I agree that `conversation_id` might fit well. As they point out: `conversation ID (sometimes called correlation ID). ` I prefer correlation here because it is more general  as we are using it to tie a link between a received message and a grpc response.